docs/ds20.md

```diff
@@ -0,0 +1,112 @@
+# fwupd BOS DS20 Specification
+
+## Introduction
+
+When `fwupd` starts it enumerates all PCI and USB hardware and generates *Instance IDs* based on the reported vendor and product so that it can match the device to a plugin.
+The plugin knows how to communicate with the device (for instance using vendor-specific USB control transfers) and also knows how to parse the firmware and deliver it to the device.
+
+Before a vendor can ship a firmware on the LVFS to a machine using Linux (or ChromeOS) the fwupd package must be updated so that it knows what plugin to use for that specific device.
```

Comment:
```suggestion
Before a vendor can ship a firmware on the LVFS to a machine using Linux (or ChromeOS) the fwupd package often must be updated so that it knows what plugin to use for that specific device.
```

Reply:
Fixed.